downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -218,6 +218,33 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced Avro serialization/deserialization support.
 
+#### `persist` package
+- Introduced support for the PostgreSQL data store, mirroring the functionality provided for other supported SQL data stores like MySQL and MSSQL.
+- Implemented support for the Redis data store, including the following features:
+  - Support for optional fields to be defined in the data model, providing flexibility in structuring data.
+  - Support for connection configuration to be defined as separate parameters or as a URI
+  - Support for Redis data store to be used as a cache or persistent store.
+
+  >**Info:** The Redis database support is an experimental feature. APIs might change in future releases.
+
+- Added support for the following annotations within the `persist.sql` package to facilitate entity mapping alongside additional SQL database features.
+  - `@sql:Name` - Map an entity name to a specific table name and a field name to a specific column name.
+  - `@sql:Varchar` - Give a specific VARCHAR length. 
+  - `@sql:Char` - Give a specific CHAR length. 
+  - `@sql:Decimal` - Give specific DECIMAL precision and scale. 
+  - `@sql:Index` - Declare an index field.
+  - `@sql:UniqueIndex` - Declare a unique index field.
+  - `@sql:Relation` - Declare a relation field. This is used to define a foreign key relationship between two entities.
+  - `@sql:Generated` - Declare a generated field. This is used to define a field that is generated by the database.
+
+#### `uuid` package
+- Implemented support for generating random UUIDs in an intuitive manner via the `uuid:createRandomUuid` function.
+
+### Improvements
+
+#### `mysql` package
+- Specified SSL as the preferred option when users provide options without SSL configuration. Additionally, introduced support for explicitly disabling SSL.
+
```

Comment:
This is incorrect, right? We already have an Improvements section starting L260. Shall we fix it? Seems like this section is now before some new features added in other PRs also.

Reply:
sure, I will move it